Example/ConsentViewController_ExampleTests/SPClientCoordinator/SourcePointClient/SourcePointClientSpec.swift

```diff
@@ -78,6 +79,18 @@ class SourcePointClientSpec: QuickSpec {
                 let expectedUrl = URL(string: "https://cdn.privacy-mgmt.com/consent/tcfv2/consent/v3/custom?scriptType=ios&scriptVersion=\(SPConsentManager.VERSION)")!.absoluteURL
                 expect(Constants.Urls.DELETE_CUSTOM_CONSENT_URL.absoluteURL) == expectedUrl
             }
+
+            it("check if language is .Spanish then ES value is included in URL") {
+                let pmUrl = Constants.Urls.GDPR_PM_URL.appendQueryItems([
+                    "message_id": "111",
+                    "pmTab": "",
+                    "consentUUID": "xxxxxx-xxxx-xxxx-8xxxxad-xxxxxxx",
+                    "idfaStatus": "accepted",
+                    "site_id": "111",
+                    "consentLanguage": SPMessageLanguage.Spanish.rawValue
+                ])
+                expect(pmUrl?.absoluteString) == "https://cdn.privacy-mgmt.com/privacy-manager/index.html?consentLanguage=ES&consentUUID=xxxxxx-xxxx-xxxx-8xxxxad-xxxxxxx&idfaStatus=accepted&message_id=111&pmTab=&site_id=111"
+            }
```

Comment:
@Nevazhnovu I believe the test above doesn't fully test the feature. 

What we're aiming for is:
1. GIVEN the user sets `.messageLanguage` to `.Spanish` on `SPConsentManager` object
2. AND the user calls `loadGDPRPrivacyManager` (or the CCPA counter part)
3. THEN the loaded URL includes the query param `consentLanguage=ES`